### PR TITLE
Add pull_request model

### PR DESCRIPTION
**Pull request by andreydro**

### CODE REVIEW

File: test/models/pull_request_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PullRequestTest < ActiveSupport::TestCase
```
Comment (kavalenka) [inline test/models/pull_request_test.rb]: Do we test smth here?)

File: test/fixtures/pull_requests.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (kavalenka) [inline test/fixtures/pull_requests.yml:1]: Let's add and configure rspec gem for testing. Please prepare a new branch after this'd be merged.

**Review by kavalenka — CHANGES_REQUESTED** [review]

File: test/fixtures/pull_requests.yml (hunk 2 above)
Comment (andreydro) [reply test/fixtures/pull_requests.yml:1]: Have I done right?

Comment (kavalenka) [reply test/fixtures/pull_requests.yml:1]: Yep, but this should be done after merging your previous changes.
Also @sergiiss has already done this in his PR.
So, please, revert your changes.

**Comment by kavalenka** [comment]
Great! Let's squash commits!

**Pull request closed without merging**